### ISSUE TITLE
confused data yaml

### ISSUE THREAD

**Issue by sipie800** [ultralytics/ultralytics#3709]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

how can I set right the data attribute here?

yolo.train(data="xxx.yaml",...)

say I've data root in D:/.../data, which contains both images and labels txt.
And the xxx.txt which specify the samples used.
How should write into xxx.yaml
I've tried a lot of things. And there is a settings.yaml which wants me to fix the data root.
And the train just tells me can't find the data or something blahblah...

So many yaml and txt, I feel really confused. Help please. 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@sipie800 just run an example training with coco8 and use that as an example to create your own datasets, i.e.:

```python
from ultralytics import YOLO

# Load a model
model = YOLO('yolov8n.pt')  # load a pretrained model (recommended for training)

# Train the model
model.train(data='coco128.yaml', epochs=3, imgsz=640)
```

Everything will install and download automatically.
